core/src/components/header/header.tsx

```diff
@@ -38,15 +40,15 @@ export class Header implements ComponentInterface {
 
   /**
    * Describes the scroll effect that will be applied to the header.
-   * Only applies in iOS mode.
+   * Only applies when the team is `"ios"`.
```

Comment:
```suggestion
   * Only applies when the theme is `"ios"`.
```

Reply:
🤣 been typing `theme` so much, that `team` looked too similar 😭

Reply:
Addressed here: https://github.com/ionic-team/ionic-framework/pull/29132/commits/caa5490a5679a0357dea81d3b477c1af5322ffba